docs/src/_reference/v2-migration.md

```diff
@@ -9,31 +9,73 @@ weight: 10
 
 ## Recommended
 
-### Migrate to an adapter-specific dbt plugin
+### Migrate to an Adapter-Specific `dbt` Transformer
 
-We recommend re-adding an adapter-specific version of dbt for existing projects which may be using the legacy `dbt` plugin name.
+If you previously used the `dbt` Transformer, we recommend migrating to an adapter-specific installation as per the `dbt` [available adapters documentation](https://docs.getdbt.com/docs/available-adapters).
 
-## Removed
+#### Install `dbt`
+
+This is easy to do! Following the instructions from above to discover and install your chosen adapter:
+
+```bash
+# list available transformer plugins
+meltano discover transformers
+
+# install adapter-specific dbt, e.g. for snowflake
+meltano add transformer dbt-snowflake
+```
+
+#### Update your `dbt_project.yml`
 
-### `model` plugin type
+Installation of a new Transformer will introduce two important files to your `transform/` directory:
 
-This plugin type provided very basic BI capabilities using Meltano UI. This functionality has been removed in favor of 3rd party open source BI solutions.
+- A new `profiles.yml` file in `transform/profiles/<adapter name>/profiles.yml`
+- A new `dbt_project.yml` file in `transform/dbt_project (<adapter name>).yml`
 
-### `dashboard` plugin type
+The new `profiles.yml` will only be used by adapter-specific `dbt` executions (e.g. `dbt-snowflake`), and can be customized to meet your requirements.
+Your existing `profiles.yml` will remain in use by your existing `dbt` Transformer plugin (via `elt` and `invoke`).
 
-This plugin type provided very basic BI capabilities using Meltano UI. This functionality has been removed in favor of 3rd party open source BI solutions.
+It is likely that the new `dbt_project (<adapter name>).yml` will contain changes from your previous `dbt_project.yml` file, especially if you haven't already upgraded to [`dbt` v1.0](https://docs.getdbt.com/guides/migration/versions/upgrading-to-v1.0).
+To complete your migration, consolidate `dbt_project.yml` and `dbt_project (<adapter name>).yml` into a single file called `dbt_project.yml`.
+As this project file will be used by both `dbt` and `dbt-<adapter>` Transformer plugins by default, you must ensure you are running an up-to-date installation of plugin `dbt` if you intend to use both adapter-specific and legacy `dbt` installs together (not recommended).
 
-### `transform` support in `meltano elt`\*\*
+If you make use of [Transform](/guide/transformation) plugins, these will continue to work as regular `dbt` packages. However adding new Transform plugins will currently (tracking at [#3304](https://github.com/meltano/meltano/issues/3304)) re-add the legacy `dbt` Transformer plugin.
+To avoid this we recommend adding Transforms as regular packages directly via the dbt CLI as per the [`dbt` Packages documentation](https://docs.getdbt.com/docs/building-a-dbt-project/package-management).
```

Comment:
```suggestion
To avoid this we recommend adding Transforms as regular packages directly via dbt as per the [`dbt` Packages documentation](https://docs.getdbt.com/docs/building-a-dbt-project/package-management).
```

Can't add packages via CLI unfortunately.